app/src/ui/add-repository/create-repository.tsx

```diff
@@ -468,9 +467,11 @@ export class CreateRepository extends React.Component<
             <Checkbox
               label="Initialize this repository with a README"
               value={
-                this.state.createWithReadme
-                  ? CheckboxValue.On
-                  : CheckboxValue.Off
+                this.state.createWithReadme ? (
+                  CheckboxValue.On
+                ) : (
+                  CheckboxValue.Off
+                )
```

Comment:
This isn’t the way the latest Prettier formats it. Are you sure you’re on the latest version?